### ISSUE TITLE
AnalyticalTable: RTL issues

### ISSUE THREAD

**Issue by Neeeko** [SAP/ui5-webcomponents-react#460]
**Describe the bug**
Not sure if that should be considered a feature request or a bug.
I found two small issues while trying to handle RTL using the AnalyticalTable.
Basically what I did was to reverse the array passed to the `columns` prop so that the column order becomes RTL compliant and I added additional classes to the Column definition to adapt the `text-align` and `justify-content` values.

However, when the `selectionMode` is set to `MULTI_SELECT`, the selection column with the checkboxes is ALWAYS stuck on the left-end side of the table (while it should be on the right side in RTL mode).
Moreover, some CSS classes are added by the library on the last column.
For example here: https://github.com/SAP/ui5-webcomponents-react/blob/master/packages/main/src/components/AnalyticalTable/hooks/useTableCellStyling.ts#L55
The thing is that in RTL mode, _the last column is actually the first one_. So the `lastColumnId` declaration in the file above is actually wrong in the RTL case.

**Expected behavior**
Ideally, the AnalyticalTable could have a RTL prop (or look for the `dir` attribute on the `html` tag), as in most use-cases, Table in SAP projects should be RTL compliant.
Basically, the AnalyticalTable in RTL mode should mirror the AnalyticalTable in LTR mode.

**Screenshots**
<img width="324" alt="Screen Shot 2020-04-28 at 19 30 20" src="https://user-images.githubusercontent.com/7316793/80518537-dc211500-8986-11ea-80e3-f5eaf7b807a9.png">
_In RTL mode, this extra padding should be applied on the left column, not the right one._

**UI5 Web Components for React Information**
Operating System:
`@ui5/webcomponents-react` version: 0.9.0-rc.16
Do you have a dependency to`@ui5/webcomponents` in your package.json? If yes, which version?: 0.0.0-185851a98

Thank you!!

**Comment by MarcusNotheis** [comment]
Released in `@ui5/webcomponents-react@0.9.0-rc.18`. RTL is inherited from the UI5 Web Components config. You can set it as described [here](https://github.com/SAP/ui5-webcomponents#configure-rtl). If you don't set it will be inherited from the language.
